core/api/core.api

```diff
@@ -238,17 +230,17 @@ public final class com/revenuecat/purchases/kmp/PurchasesConfiguration$Builder {
 	public final fun getAppUserId ()Ljava/lang/String;
 	public final fun getDangerousSettings ()Lcom/revenuecat/purchases/kmp/DangerousSettings;
 	public final fun getDiagnosticsEnabled ()Z
-	public final fun getPurchasesAreCompletedBy ()Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;
+	public final fun getPurchasesAreCompletedBy ()Lcom/revenuecat/purchases/PurchasesAreCompletedBy;
 	public final fun getShowInAppMessagesAutomatically ()Z
 	public final fun getStore ()Lcom/revenuecat/purchases/Store;
 	public final fun getUserDefaultsSuiteName ()Ljava/lang/String;
 	public final fun getVerificationMode ()Lcom/revenuecat/purchases/kmp/EntitlementVerificationMode;
-	public final fun purchasesAreCompletedBy (Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;)Lcom/revenuecat/purchases/kmp/PurchasesConfiguration$Builder;
+	public final fun purchasesAreCompletedBy (Lcom/revenuecat/purchases/PurchasesAreCompletedBy;)Lcom/revenuecat/purchases/kmp/PurchasesConfiguration$Builder;
 	public final synthetic fun setApiKey (Ljava/lang/String;)V
 	public final synthetic fun setAppUserId (Ljava/lang/String;)V
 	public final synthetic fun setDangerousSettings (Lcom/revenuecat/purchases/kmp/DangerousSettings;)V
 	public final synthetic fun setDiagnosticsEnabled (Z)V
-	public final synthetic fun setPurchasesAreCompletedBy (Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;)V
+	public final synthetic fun setPurchasesAreCompletedBy (Lcom/revenuecat/purchases/PurchasesAreCompletedBy;)V
```

Comment:
This is not a breaking change afaik, as `PurchasesAreCompletedBy` is now a type alias. It looks like the `.api` file contains the actual underlying type instead of the alias, which makes sense I guess.

Reply:
Hmm I wonder..., in this case we wouldn't be testing the ios API, which is actually `com.revenuecat.purchases.kmp.PurchasesAreCompletedBy` right? Seems like the public API should use the `kmp` version, like for example the `EntitlementVerificationMode` enum... Not sure of the reason for this change though...

Reply:
The iOS API is in a different file (`core.klib.api`) 🙂 That file specifies the `.kmp.` enum indeed, see [here](https://github.com/RevenueCat/purchases-kmp/blob/26245aab0fb822bc9226cf3626ea923828540fd6/core/api/core.klib.api#L214).

The reason why `EntitlementVerificationMode` shows up as `.kmp.` even in the JVM API (`core.api`) is because it is not a type alias. So the only type is the `.kmp.` type.  I'm not sure from the top of my head why it's not a type alias. Maybe because we never need to convert it anyway.